ext/posix/posix.c

```diff
@@ -713,6 +713,44 @@ PHP_FUNCTION(posix_access)
 
 	RETURN_TRUE;
 }
+
+#ifdef HAVE_EACCESS
+PHP_FUNCTION(posix_eaccess)
+{
+	zend_long mode = 0;
+	size_t filename_len, ret;
+	char *filename, *path;
+
+	ZEND_PARSE_PARAMETERS_START(1, 2)
+		Z_PARAM_PATH(filename, filename_len)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_LONG(mode)
+	ZEND_PARSE_PARAMETERS_END();
+
+	path = expand_filepath(filename, NULL);
+	if (!path) {
+		zend_argument_value_error(1, "cannot be empty");
```

Comment:
I think our standard writing is as such. @kocsismate is this correct?
```suggestion
		zend_argument_value_error(1, "must not be empty");
```

Reply:
No, the wording is somehow `cannot be empty` :) We do have a couple of `must not be empty` as well, but it's much less than the former one